contributor-docs/testing.md

```diff
@@ -0,0 +1,128 @@
+# Testing
+
+<!-- prettier-ignore-start -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Table of Contents
+
+- [Overview](#overview)
```

Comment:
Thanks for the great documentation! It helped a lot when setting it up on my local ✨